templates/generic/tag_multiselect.html

```diff
@@ -1,8 +1,11 @@
 <div>
-  <select data-info="{{ object.object_info }}" class="tag-multiselect" name="state[]" multiple placeholder="Select a tag..." autocomplete="off">
-    {% for tag in available_tags %}
-      <option value="{{ tag }}" {% if tag in object.get_linked_tags %}selected{% endif %}>{{ tag }}</option>
-    {% endfor %}
-  </select>
+    <select data-info="{{ object.object_info }}" class="tag-multiselect" name="state[]" multiple placeholder="Select a tag..." autocomplete="off">
+        {% for tag in available_tags %}
+            {% if tag not in system_tags %}
```

Comment:
You could instead just do {% if not tag.is_system_tag %}, and pass the actual tag objects into the template. 

Otherwise, I think if you have both a system tag and a regular tag with the same name, you won't be able to edit the regular tag.

Reply:
What is the utility of showing system tags in the dropdown?

Reply:
ha I guess not much utility-- I removed all together from drop downs but it will stay in the Tabs table (but still cant be edited) [6412ba3](https://github.com/dimagi/open-chat-studio/pull/440/commits/6412ba37b10b0637bd6fd219aec27d9f3d2e486d)